x/fees/types/min_fee.go

```diff
@@ -0,0 +1,37 @@
+package types
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// MinFee contains the minimum amount of coins that should be paid as a fee for each message of the specified type sent
+type MinFee struct {
+	MessageType string  `json:"message_type" yaml:"message_type"`
+	Amount      sdk.Dec `json:"amount" yaml:"amount"`
```

Comment:
Instead of using `sdk.Dec` here, I think the best thing to do is to use `sdk.Coins` instead. This way, we can even set different fee coins based on a message type: 

```json
[
  {
    "message_type": "create_validator",
    "amount": {
      "amount": "200",
      "denom": "udaric"
    }
  },
  {
    "message_type": "edit_validator",
    "amount": {
      "amount": "100",
      "denom": "upotic"
    }
  }
]
```